package/Config.in

```diff
@@ -7,6 +7,7 @@ menu "Target packages"
 	source "package/skeleton-init-none/Config.in"
 	source "package/skeleton-init-systemd/Config.in"
 	source "package/skeleton-init-sysv/Config.in"
+	source "package/motioneye/Config.in"
```

Comment:
I don't think adding `motioneye` package here is a good idea, since motionEye is far from a regular BuildRoot package in `motionEyeOS`. It simply won't work on anything else than `motionEyeOS`.